app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics.controllers.js");
+
+app.use(express.json())
```

Comment:
This should only be here when we're posting, you don't need this for get requests.